CHANGES.md

```diff
@@ -1,6 +1,417 @@
 Release Changes List
 ====================
 
+*** stopped at dced642 2017-09-24 ***
+
+TODO: move docs to top-level? ('latest' is symbolic link to '.'?)
+TODO: remove 'master'/'latest' from any docs links?
+TODO: where to put LLVM codegen flags in 6135 / 8d704f0
+TODO: error-handling keyword highlighting in emacs
+TODO: 'tics' vs. `backtics`?
+TODO: refactor / filter highlights
+TODO: reorder sections
+
+version 1.16.0
+==============
+
+Nineteenth public release of Chapel, September 20, 2017
+
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* added a new 'configure'/'make install' option for building+installing Chapel
+* added prototypical support for 'Mason' v0.1.0, Chapel's package manager
+* added a new c2chapel tool that converts C headers to Chapel extern decls
+* 'chpl'-generated executables now take the main module name rather than 'a.out'
+* added a 'defer' statement to aid with cleanup
+* added support for 'try'/'try!' expressions (no 'catch' permitted)
```

Comment:
Maybe
* added support for 'try'/'try!' expressions ('catch' only on statement form)